pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -645,8 +643,8 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 				sysAdapter.Name, isMgmt, isFree,
 				sysAdapter.Uplink, sysAdapter.FreeUplink)
 			// Either one can set isMgmt; both need to clear
-			if sysAdapter.Uplink && !isMgmt {
-				log.Warnf("Mgmt flag forced by system adapter for %s",
+			if sysAdapter.Uplink {
+				log.Warnf("Mgmt flag set by system adapter for %s",
```

Comment:
Should be a log.Infof

Reply:
But actually I'd write the code as for isFree i.e., 
isMgmt = sysAdapter.Uplink
log.Infof()